Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM ubuntu:latest
+RUN apt update
+# Avoid prompt for locale.
+RUN DEBIAN_FRONTEND="noninteractive" apt install -y tzdata
+RUN apt install -y git netcat wget
+RUN apt install -y openjdk-11-jdk maven
```

Comment:
As mentioned above, you should consider using the `maven:3.6.3-openjdk-11` base image for the build and `openjdk:11` for the resulting image.